tools/ciscospark.md

```diff
@@ -0,0 +1,26 @@
+## Connecting Rollbar to Cisco Spark
+
+For general information about Rollbar's alerting features, check out the [Notifications guide](../notifications/). 
+{: .info}
+
+### Setup
+
+Configuration is per-project in Rollbar.
+
+1.  Head to the Notification settings page for a project: Dashboard ->
+    Settings -> Notifications -> Cisco Spark.
+2.  Click **Connect with Cisco Spark**
+3.  Select the Cisco account you want to integrate with, and authorize the
+    permission request.
+4.  Select the room you wish to receive messages for
+5.  Click **Enable Cisco Spark Integration**
+6.  Congrats! Cisco Spark is now integrated with Rollbar. Default rules will
```

Comment:
I would change the wording to 'Default rules have been' instead of 'will have been'